### PR TITLE
MNT: Re-rendered with conda-build 3.18.9, conda-smithy 3.6.0, and con…

### PR DESCRIPTION
**Pull request by jklynch**
…da-forge-pinning 2019.10.28

http://yapsy.sourceforge.net

### CODE REVIEW

**Comment by mrakitin** [comment]
The name should be upper-case:
image

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
   version: "{{ version }}"
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|capitalize }}-{{ version }}.tar.gz"
```
Comment (mrakitin) [inline recipe/meta.yaml]: I'd better recommend to change the `{% set name = "yapsy" %}` on line 1 to have `Yapsy` there. The `name` var is already lowercased.

Comment (jklynch) [reply recipe/meta.yaml]: The directories {{ name[0] }} and {{ name }} are not capitalized so those would have to become {{ name[0]|lower }} and {{ name|lower }}.

Comment (mrakitin) [reply recipe/meta.yaml]: Interesting. It seems to work for `ModestImage`: https://github.com/nsls-ii-forge/modestimage-feedstock/blob/4da15e709d1be189ddfa1e060e5379bbc2c84827/recipe/meta.yaml#L10.

Comment (jklynch) [reply recipe/meta.yaml]: I was confused by the duplicated "name" variables. The duplication of variables by the {% set... %} lines and the yaml seems unnecessary. What is the advantage?

Comment (mrakitin) [reply recipe/meta.yaml]: The `{% set name = ... %}` variable is used by jinja to expand the names in the template.
The `package: --> name: ...` is used by conda to name the resulted package. And in this case it's also templated.

In other words, the `{% set ... %}` things do not appear in the final recipe after its rendering by jinja. Here is an example of the modestimage package's contents:
image

**Pull request merged**